gqcp/include/QCMethod/CC/CCD.hpp

```diff
@@ -60,7 +60,7 @@ class CCD {
         // Furthermore, the solvers only find the ground state wave function parameters, so the QCStructure only needs to contain the parameters for one state.
         const auto& T2 = environment.t2_amplitudes.back();
 
-        const auto E_electronic_correlation = environment.electronic_energies.back();
+        const auto E_electronic_correlation = environment.correlation_energies.back();
```

Comment:
This name is much clearer and reflects the purpose of this variable in a better way.